client/main.css

```diff
@@ -1411,13 +1466,13 @@ audio {
   .navbar-toggle {
     margin-right: 5px;
   }
-  .footer__links { 
-    float: none; 
+  .footer__links {
+    float: none;
     width: 100%;
     margin-bottom: 20px;
   }
   .footer .footer__links:nth-child(2) .footer__subheading {
-    display: none; 
+    display: none;
```

Comment:
All the CSS is looking solid. We can continue to tweak it as needed. Thanks for cleaning up the extra trailing spaces.